### DISCUSSION TITLE
Allow us a way to donate?

### DISCUSSION THREAD

**Discussion by maxichrome** [twpayne/chezmoi#1811] [General]
Hey Tom & chezmoi maintainers!

I think it would be super cool for y'all to open a GitHub Sponsors page for chezmoi. For how convenient, useful, nicely-documented it is, it would be awesome to be able to contribute monetarily — even if just a few bucks a month, for your work! Maintaining an open-source project, [as we know](https://youtu.be/d15DP5zqnYE), can be a thankless & undervalued job at times.

Of course, this is just a discussion, so feel free to add any thoughts or feedback. Regardless of any outcome, thank you for all you do! 💜

**Comment by twpayne** [comment]
Thanks for the kind words!

I'm not planning to enable GitHub sponsorships for chezmoi at this point for a variety of reasons. If you find chezmoi useful, please make a donation to a charity of your choice. Thank you!

**Reply by zhangtai** [reply]
Have no idea which charity to donate 🤣 would you plz enable the donation, even you don't need the money, donate to who you believe needs it, I believe you can make a good decision.

**Reply by twpayne** [reply]
That's very kind, but part of the reason that I don't suggest a charity is that I want to focus on the code and don't want to be involved with the administrative or political aspects of charitable donations.
